tests/test_plugin.py

```diff
@@ -29,12 +32,16 @@
 
 
 def test_load_prebuild_plugins():
+    if MOCK:
+        mock_docker()
     runner = PreBuildPluginsRunner(DockerTasker(), DockerBuildWorkflow(SOURCE, ""), None)
```

Comment:
Why not use the `docker_tasker` fixture here, and in `test_load_postbuild_plugins()` below?